src/Samhammer.AzureBlobStorage/StreamHelper.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Microsoft.IO;
+
+namespace Samhammer.AzureBlobStorage
+{
+    public class StreamHelper
+    {
+        private const long DefaultMaxSmallPoolFreeBytes = 1000000;
+        private const long DefaultMaxLargePoolFreeBytes = 10000000;
+        private static string maxSmallPoolFreeBytes = Environment.GetEnvironmentVariable("MAX_SMALLPOOL_FREEBYTES");
```

Comment:
@khoanguyen102 i think it would be better to hav configrations loaded from IConfiguration/IOptions instead of only support enviroment variables, as we can not configure different values per appsettings otherwise.

Reply:
Also we should add some short note about the settings to the Readme.

Reply:
Thanks Roli, I have updated it and also readme